### ISSUE TITLE
Training questions

### ISSUE THREAD

**Issue by kevinjohncutler** [MouseLand/cellpose#131]
The documentation is a bit sparse on the retraining process. I have a few questions that I think would help other new users such as myself. 

1. Could you provide an example for running training via Jupyter Notebooks? 
2. What is the recommended number and size of images for training?
3. What criteria should decide when the model is well-trained / what parameters should I try tuning to maximize performance?

With regard to 1, I looked at the API and it seems like the only way to use the train() function is to manually load in all the images, masks, etc. and pass them in. I was hoping there was a cleaner way with a directory like the CLI implementation. 

With regard to 2, my own training set is made up of four 512x512 grayscale images with ~400 cells each and their corresponding labels (I split the original 2048x2048 dataset into 16 smaller images that were more convenient to segment manually).  It might be the case that I need to split up the images further, but cellpose takes 224x224 subsections and that would mean that each of these images would have ~100 cells as the documentation seems to suggest is optimal.

**Comment by carsen-stringer** [comment]
Do you have an example for (1) since you've been training models? We are happy to take pull requests since we usually train via the CLI. For (2) we use 224x224 patches so there is no optimal size. We resize so that the diameter of masks is on average 30 pixels (if you are using the cyto model) so I'd just make sure they are bigger than 224x224 after that resizing. For (3) we don't have a good sense since we haven't done much transfer learning. For the full model we train on 540 images for about 500 epochs and see the training and test loss saturate by then.

**Comment by kevinjohncutler** [comment]
Thanks for your reply Carsen. I also have been doing all the training via CLI, but starting training via jupyter could be more convenient... however, I just realized that one can simply pass a system command from a notebook (just as in MATLAB) so that problem is solved!
